app/src/components/misc/PersonaSurvey/config.tsx

```diff
@@ -1,13 +1,10 @@
-import { PageConfig, PersonaType, QuestionnaireType, SurveyOptionsConfig } from "./types";
+import { PageConfig, PersonaType, QuestionnaireType, SurveyOptionsConfig, SurveyPage } from "./types";
 import { setUserPersona } from "./actions";
 import { GettingStartedWithSurvey } from "./GettingStartedWithSurvey";
 
 export const OptionsConfig: Record<QuestionnaireType, SurveyOptionsConfig> = {
   [QuestionnaireType.PERSONA]: {
-    key: "persona",
-    questionType: "single",
-    isActive: ({ key, title }) => key === title,
-    action: (dispatch, value, clear) => setUserPersona(dispatch, value, clear, "persona"),
+    action: (dispatch, value, doClear) => setUserPersona(dispatch, value, doClear, QuestionnaireType.PERSONA),
```

Comment:
can we please rename `value` and `doClear` parameters or add a doc comment about the parameters in type definition